src/ui/pointcloud/qgspointcloudelevationpropertieswidgetbase.ui

```diff
@@ -128,6 +128,136 @@
      </layout>
     </widget>
    </item>
+   <item>
+    <widget class="QGroupBox" name="mLayerRenderingGroupBox">
+     <property name="title">
+      <string>Profile Chart Accuracy</string>
```

Comment:
@wonder-sk something I was toying with was just reusing that setting from the 2d renderer. What do you think?

Reply:
I thought about that too, but I am undecided. It would be nice though if user needed to adjust fewer knobs... So maybe it would make sense to default to reusing _all_ the settings from 2D renderer (point styling/color + size + shape + error), and then keep the option to override the settings (what we have now).